instrumentation/p6spy/src/main/java/brave/p6spy/TracingJdbcEventListener.java

```diff
@@ -30,6 +34,7 @@
     // regardless of noop or not, set it in scope so that custom contexts can see it (like slf4j)
     if (!span.isNoop()) {
       String sql = includeParameterValues ? info.getSqlWithValues() : info.getSql();
+      if (sql == null) sql = QUERY_PLACEHOLDER;
       span.kind(Span.Kind.CLIENT).name(sql.substring(0, sql.indexOf(' ')));
```

Comment:
if you use tertiary op here, you don't need to play with extra span.

Ex. scrub the constant and 

```java
// use "query" if p6spy wasn't able to intercept the SQL.
.name(sql == null ? "query" : sql.substring(0, sql.indexOf(' ')));
```